src/utils.js

```diff
@@ -705,6 +705,7 @@ export const unshift = (array: Array<any>, item: any): any => {
 export const createAddPropertiesToFunction = (cache: Cache, originalFunction: Function, options: Options) => {
   return (moizedFunction: Function): Function => {
     const getCacheKey = createGetCacheKey(cache, options);
+    const setNewCachedValue = createSetNewCachedValue(cache, options);
```

Comment:
this allows the `add` method to use the same options that apply to the `cache` when manually adding things to cache.